pages/common/gpg-zip.md

```diff
@@ -0,0 +1,15 @@
+# gpg-zip
+
+> Encrypt files and directories in an archive using GPG.
+
+- Encrypt a directory into archive.gpg using a passphrase:
+
+`gpg-zip --symmetric --output {{archive.gpg}} {{dirname}}`
+
+- Decrypt archive.gpg into a directory of the same name:
+
+`gpg-zip --decrypt {{archive.gpg}}`
+
+- List contents of the encrypted archive.gpg:
```

Comment:
Perhaps `List the contents`.... would read better? Also `archive.gpg` -> `path/to/archive.gpg`

Reply:
Thanks for the suggestions, I have incorporated most of them. Just wantedd to clarify if I should change "List contents of the encrypted archive.gpg" to:

1. List the contents of the encrypted archive.gpg
2. List the contents of archive.gpg
3. List the contents

Which one of the above reads better?

Reply:
Number 1 of that list I think :slightly_smiling_face: